### PR TITLE
Rda 1331/new schema

### PR DESCRIPTION
**Pull request by nickpricedev**
Changes to ingestion pipeline for new Codehub schema

### CODE REVIEW

**Review by Rodriguez-Carlos-BAH — APPROVED** [review]
Approved

File: ingest/ingest.py [hunk 1]
```diff
@@ -18,162 +18,247 @@
...
+    result = (repo['sourceData']['stars']*3) + (repo['sourceData']['watchers']*4) + (len(repo['sourceData']['contributors'])*5) + repo['sourceData']['commits']
+    return result
+
+def cloneGithubRepo(repo):
+    ownerName = repo['sourceData']['owner']['name']
+    repoName = repo['sourceData']['name']
+
+    rootdir = expanduser("~") + '/cloned_projects'
+    repodir = rootdir + '/' + ownerName + '/' + repoName
+    # if os.path.exists(repodir):
```
Comment (mvs5465) [inline ingest/ingest.py]: Lets remove unused code or add a comment to why it is commented out

**Review by mvs5465 — APPROVED** [review]
I think it would be helpful to break this file apart, separating ingest-specific things from generic stuff. Its getting pretty big.

Otherwise I added minor comment for commentary about commenting comments.

Approved. Looks good, nice work.

File: ingest/ingest.py (hunk 1 above)
Comment (nickpricedev) [reply ingest/ingest.py]: Done

**Pull request merged**